project/Game.py

```diff
@@ -137,7 +149,15 @@ def quit_question(self, team, password):
         return True
 
     def answer_question(self, team, answer):
-        return True
+        if self.game.landmarks[team.currentLandmark].answer is not answer:
+            self.team.penalty_count += 1
```

Comment:
I think I misspoke here. I was thinking we were in Team. This should call the team function, but there's no self.team. looks like the team is being passed in, so it could be team.setPenalty()